proposals/simd/NewOpcodes.md

```diff
@@ -0,0 +1,124 @@
+| i8x16.shr_s          | 0x6c   | i16x8.shr_s              | 0x8c   | i32x4.shr_s              | 0xac   | i64x2.shr_s | 0xcc   |
+| i8x16.shr_u          | 0x6d   | i16x8.shr_u              | 0x8d   | i32x4.shr_u              | 0xad   | i64x2.shr_u | 0xcd   |
+| i8x16.add            | 0x6e   | i16x8.add                | 0x8e   | i32x4.add                | 0xae   | i64x2.add   | 0xce   |
+| i8x16.add_saturate_s | 0x6f   | i16x8.add_saturate_s     | 0x8f   | ---- add_sat ----        | 0xaf   | ----        | 0xcf   |
+| i8x16.add_saturate_u | 0x70   | i16x8.add_saturate_u     | 0x90   | ---- add_sat ----        | 0xb0   | ----        | 0xd0   |
+| i8x16.sub            | 0x71   | i16x8.sub                | 0x91   | i32x4.sub                | 0xb1   | i64x2.sub   | 0xd1   |
+| i8x16.sub_saturate_s | 0x72   | i16x8.sub_saturate_s     | 0x92   | ---- sub_sat ----        | 0xb2   | ----        | 0xd2   |
+| i8x16.sub_saturate_u | 0x73   | i16x8.sub_saturate_u     | 0x93   | ---- sub_sat ----        | 0xb3   | ----        | 0xd3   |
+| ---- dot ----        | 0x74   | ---- dot ----            | 0x94   | i32x4.dot_i16x8_s        | 0xb4   | ----        | 0xd4   |
+| ---- mul ----        | 0x75   | i16x8.mul                | 0x95   | i32x4.mul                | 0xb5   | i64x2.mul   | 0xd5   |
+| i8x16.min_s          | 0x76   | i16x8.min_s              | 0x96   | i32x4.min_s              | 0xb6   | ----        | 0xd6   |
+| i8x16.min_u          | 0x77   | i16x8.min_u              | 0x97   | i32x4.min_u              | 0xb7   | ----        | 0xd7   |
+| i8x16.max_s          | 0x78   | i16x8.max_s              | 0x98   | i32x4.max_s              | 0xb8   | ----        | 0xd8   |
+| i8x16.max_u          | 0x79   | i16x8.max_u              | 0x99   | i32x4.max_u              | 0xb9   | ----        | 0xd9   |
+| ---- avgr_s ----     | 0x7a   | ---- avgr_s ----         | 0x9a   | ---- avgr_s ----         | 0xba   | ----        | 0xda   |
+| i8x16.avgr_u         | 0x7b   | i16x8.avgr_u             | 0x9b   | ---- avgr_u ----         | 0xbb   | ----        | 0xdb   |
+
+| f32x4 Op        | opcode | f64x2 Op        | opcode |
+| --------------- | ------ | --------------- | ------ |
+| f32x4.abs       | 0xe0   | f64x2.abs       | 0xf0   |
+| f32x4.neg       | 0xe1   | f64x2.neg       | 0xf1   |
+| ---- round ---- | 0xe2   | ---- round ---- | 0xf2   |
+| f32x4.sqrt      | 0xe3   | f64x2.sqrt      | 0xf3   |
+| f32x4.add       | 0xe4   | f64x2.add       | 0xf4   |
+| f32x4.sub       | 0xe5   | f64x2.sub       | 0xf5   |
+| f32x4.mul       | 0xe6   | f64x2.mul       | 0xf6   |
+| f32x4.div       | 0xe7   | f64x2.div       | 0xf7   |
+| f32x4.min       | 0xe8   | f64x2.min       | 0xf8   |
+| f32x4.max       | 0xe9   | f64x2.max       | 0xf9   |
+| ---- pmin ----  | 0xea   | ---- pmin ----  | 0xfa   |
+| ---- pmax ----  | 0xeb   | ---- pmax ----  | 0xfb   |
+
+| Conversion Op           | opcode |
+| ----------------------- | ------ |
+| i32x4.trunc_sat_f32x4_s | 0x100   |
```

Comment:
nit: can this be written as 0x0100? It makes it slightly clearer that it's 2 bytes.